multi_partner_learning.py

```diff
@@ -274,7 +273,12 @@ def compute_collaborative_round_fedavg(self):
 
         # Evaluate and store accuracy of mini-batch start model
         model_to_evaluate = partners_model_list_for_iteration[0]
-        model_evaluation = model_to_evaluate.evaluate(x_val, y_val, batch_size=constants.DEFAULT_BATCH_SIZE, verbose=0)
+
+        if not hasattr(model_to_evaluate, 'coef_'):
+            model_evaluation = [0]*2
```

Comment:
I don't understand why we need this bit

Reply:
It's needed for non fitted model evalutation which keras does support, but sklearn doesn't.

I move it to `evaluate_model`